### DISCUSSION TITLE
Can I add an additional custom field like Discription to a To-Do

### DISCUSSION THREAD

**Discussion by witzker** [Leantime/leantime#649] [Project Management]
Hi
I just installed leantime
Is there a way to customize the To-dos that they can look different with custom fields ?

e.g. one for To-do with different fields than
one for Bugs and so on like e.g in openprojekt (where different kinds of tasks are  possible)?
https://www.openproject.org/de/

THX for advices
additional

**Comment by marcelfolaron** [comment]
That is currently not possible. But feel free to add a feature request for
that.
